### PR TITLE
Math magician test

### PR DESCRIPTION
**Pull request by iankiema**
Added the following test files
-Calculate.test.js
-Operate.test.js
-Calculator.test.js
Quote.test.js

### CODE REVIEW

File: src/components/Calculator.test.js
Comment (aamirkhan2478) [inline src/components/Calculator.test.js:1]: - [ ] Please write snap tests for all components for now you just write a snap test for the Quote component kindly add snap tests for the Calculator and Button components as well :slightly_smiling_face:

File: src/components/Quote.test.js
Comment (aamirkhan2478) [inline src/components/Quote.test.js:1]: - [ ] Please fix some errors which come from the `Quote.test.js` file check below :point_down: the errors and kindly fix it :slightly_smiling_face:  

Screenshot from 2023-08-22 15-43-21

Screenshot from 2023-08-22 15-43-34

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @iankiema & @ngalamac , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @JavierAybar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Tests are passing  ✔️ 
-Tests for quotes are presents  :heavy_check_mark: 
- Correct GitHub workflow :heavy_check_mark: 
- Fantastic commit history :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**